buildpacks/nodejs/lib/build.sh

```diff
@@ -162,6 +162,15 @@ set_node_env() {
 	fi
 }
 
+set_node_modules_env() {
+	local layer_dir=$1
+
+	mkdir -p "${layer_dir}/env"
+	if [[ ! -s "${layer_dir}/env/NODE_MODULES_ENV" ]]; then
+		echo "$(cd "$(dirname "${layer_dir}/node_modules")"; pwd -P)/$(basename "${layer_dir}/node_modules")" >>"${layer_dir}/env/NODE_MODULES_ENV"
```

Comment:
@danielleadams is this the right way to find path of node modules and copy it into NODE_MODULES_ENV?

Reply:
I think you may be able to do `echo "${layer_dir}/node_modules"`